docs/csharp/tutorials/microservices.md

```diff
@@ -48,7 +48,7 @@ source, cross platform editor. However, you can use whatever tools you are
 comfortable with.
 
 You'll also need to install the Docker engine. See the 
-[Docker Installation page](http://www.docker.com/products/docker) 
+[Docker Installation page](https://www.docker.com/products/docker) 
```

Comment:
This is no longer the installation page. Maybe link to this one instead: https://docs.docker.com/install/overview/

Reply:
Won't be better to use directly this one?
https://docs.docker.com/get-started/